using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md

```diff
@@ -149,7 +228,7 @@ Returns the visitor profile that was most recently updated. The visitor profile
 {% tab title="Android" %}
 ### getVisitorProfile
 
-In Android, the visitor profile is saved in `SharedPreferences`.
+This API returns the visitor profile that was most recently obtained. Visitor profile is saved in `SharedPreferences` for easy access across multiple launches of your app. If no signal has been submitted, null is returned.
```

Comment:
Please reword the 2nd sentence, so that it reads like this:
"For easy access across multiple launches of your app, the visitor profile is saved in `SharedPreferences`. "